tsl/test/shared/expected/constraint_aware_append.out

```diff
@@ -0,0 +1,24 @@
+-- test handling of ConstraintAwareAppend -> Sort -> Result -> Scan
+-- issue 5739
+CREATE TABLE ca_append_result(time timestamptz NULL, device text, value float);
+SELECT table_name FROM create_hypertable('ca_append_result', 'time');
+NOTICE:  adding not-null constraint to column "time"
+    table_name    
+ ca_append_result
+(1 row)
+
+ALTER TABLE ca_append_result SET (timescaledb.compress);
+INSERT INTO ca_append_result SELECT '2000-01-03','d1',0.3;
+SELECT count(compress_chunk(ch)) AS compressed FROM show_chunks('ca_append_result') ch;
+ compressed 
+          1
+(1 row)
+
+INSERT INTO ca_append_result SELECT '2000-01-13','d1',0.6;
+SET enable_seqscan TO FALSE;
+SELECT time_bucket('20d',time) AS day from ca_append_result WHERE time > '1900-01-01'::text::timestamp GROUP BY day;
+             day              
+ Sun Jan 02 16:00:00 2000 PST
+(1 row)
+
+DROP TABLE ca_append_result;
```

Comment:
Since you clean up in this test case, could you also reset `enable_seqscan`?

Reply:
Why? enable_seqscan is a session setting so it does not have to be changed back at session end, while the created table persists after the life time of this test.

Reply:
It's a nit: If more test cases are added, it might become difficult to understand which custom settings have already been made. So resetting this after the test case might be beneficial for subsequent test case authors.